docs: Revise thermocycler lid text

jwwojak opened the pull request:
# Overview

The thermocycler section in the Modules chapter is out of date because it only mentions the rubber automation seal. We've since added a disposable plastic lid. Changes here revise the intro and add information about the rubber seal and newer disposable PCR plate.

RTC-587

## Test Plan and Hands on Testing

Review for style and appropriate code use.

## Changelog

- Revised thermocycler section intro
- New H4 section that lists and describes the rubber seal and disposable pcr plate seal.

## Review requests

Review for style and appropriate code use.

## Risk assessment

Low. Docs changes only.

jwwojak committed:
Add lid information to TC section

Revise section intro

Replaces "Rubber automation seals" section with information about both seal types: the rubber seal and disposable, single-use seal.

jwwojak committed:
Small change to intro

ecormany commented on docs/flex-manual/docs/modules.md:
```diff
@@ -500,7 +500,7 @@ Outside of protocols, the Opentrons App can display the current status of the Te
 ### Thermocycler features
 
-The Opentrons Thermocycler Module GEN2 is a fully automated on-deck thermocycler, providing hands-free PCR in a 96-well plate format. Its heated lid and disposable seal fit tightly over the plate, ensuring efficient sample heating and minimal evaporation.
+The Opentrons Thermocycler Module is a fully automated on-deck thermocycler, providing hands-free PCR in a 96-well plate format. It is compatible with the Flex Gripper, other deck-mounted hardware, and supported in the Opentrons App and Python API. The heated lid and either a disposable rubber seal or reusable PCR plate lid help ensure a tight, consistent seal over your samples, which is crucial  ...
```
Tough Auto-sealing Lids are designed to be disposed after use, so we shouldn't call them "reusable".

ecormany replied:
In fact, the table below confirms this:
> These sterile, single-use PCR plate lids...

jwwojak replied:
I think this got flipped around.

- Rubber seal: reusable
- Lid: disposable, single-use

Glarrrhg. Will fix it. 😳

ecormany commented on docs/flex-manual/docs/modules.md:
```diff
@@ -522,9 +522,19 @@ The automated lid can be opened or closed as needed during protocol execution.
+| Lid Type | Description |
+|----|----|
+| Opentrons Tough PCR Auto-sealing Lid | These sterile, single-use PCR plate lids help prevent cross-contamination and evaporation during Thermocycler incubation periods. The lids are Gripper-compatible and can be stacked directly on the deck or placed in a special deck riser. |
+| Rubber Automation Seal | These are adhesive-backed Ethylene Propylene Diene Monomer (EPDM) seals you manually apply to the Thermocycler lid. Rubber seals can be reused up to 20 times; however, unlike the Opentrons Tough Auto-sealing lid, they are not sterile. The seals must be cleaned and sanitized before each use. |
```
Common and proper noun capitalization
```suggestion
| Rubber Automation Seal | These are adhesive-backed ethylene propylene diene monomer (EPDM) seals you manually apply to the Thermocycler lid. Rubber seals can be reused up to 20 times; however, unlike the Opentrons Tough Auto-sealing Lid, they are not sterile. The seals must be cleaned and sanitized before each use. |
```

ecormany commented on docs/flex-manual/docs/modules.md:
```diff
@@ -522,9 +522,19 @@ The automated lid can be opened or closed as needed during protocol execution.
+| Rubber Automation Seal | These are adhesive-backed Ethylene Propylene Diene Monomer (EPDM) seals you manually apply to the Thermocycler lid. Rubber seals can be reused up to 20 times; however, unlike the Opentrons Tough Auto-sealing lid, they are not sterile. The seals must be cleaned and sanitized before each use. |
+
+!!!warning
+    Do not use the Opentrons Tough Auto-sealing PCR Lid and a rubber automation seal on the Thermocycler at the same time. This combination prevents the module's lid from closing properly, which can cause temperature control problems and mechanical damage. Always remove the rubber seal before running protocols that use the disposable PCR lid.
```
A place for our favorite image? It could even go inside the admonition!
<img width="1418" height="945" alt="Adhesive Seal Warning" src="https://github.com/user-attachments/assets/10f49e76-ffc6-4c7c-8f5e-149c9a3d7e47" />

jwwojak replied:
oooh! let's try it. Time to dive into the rabbit hole! 🐇

jwwojak replied:
The madness, I did it.

<img width="763" height="518" alt="Screenshot 2025-07-11 at 1 36 52 PM" src="https://github.com/user-attachments/assets/f5941498-a3ac-4e9f-9980-d792b0eedfa5" />

Could be good to use as it matches the card that ships in each thermocycler box.

ecormany reviewed (commented):
Couple proofread comments.

jwwojak committed:
Update docs/flex-manual/docs/modules.md

Committing suggestion. Thank you.

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
Clarity about lid use, add warning image

The rubber seal is reuseable.

The plate lid is single-use.

Replace text in the warning callout with the blow-in card image. A card with this image ships in every thermocycler box.

Revise summary intro. Combined concepts from the website into the paragraph.

ecormany commented on docs/flex-manual/docs/modules.md:
```diff
@@ -529,12 +529,10 @@ The Thermocycler works with two different plate seals to help protect your sampl
-    Do not use the Opentrons Tough Auto-sealing PCR Lid and a rubber automation seal on the Thermocycler at the same time. This combination prevents the module's lid from closing properly, which can cause temperature control problems and mechanical damage. Always remove the rubber seal before running protocols that use the disposable PCR lid.
-
-
+    ![Do not use the rubber seal and Opentrons tough seal simultaneously.](images/thermocycler-seal-warning.png)
```
I liked the more detailed warning text here. I think we can keep both, especially with a minor edit to the image to remove the text and Opentrons logo, which is out of place in the middle of docs like this. 

Here's what that would look like in context:
<img width="715" height="519" alt="image" src="https://github.com/user-attachments/assets/77a4f94a-0654-4d4b-8c6f-2c01c6b81ec4" />

And here's the modified image you can swap in:
<img width="1356" height="708" alt="thermocycler-seal-warning" src="https://github.com/user-attachments/assets/c3e58429-7753-4ddb-9f92-52d85e17a479" />

jwwojak replied:
Updating in next push!

jwwojak committed:
Add text and image to lid warning callout

- Warning text similar to the card that ships in-box with module.

- Add revised warning image.

jwwojak reviewed (commented):
Made review comment changes.

- New lid image.
- Prose changes.

ecormany reviewed (approved):
Looks good!
